libs/sdk-ui-dashboard/src/model/state/topBarContext/topBarState.ts

```diff
@@ -0,0 +1,9 @@
+// (C) 2021 GoodData Corporation
+
+export interface TopBarState {
+    title: string;
```

Comment:
The title should come from the `meta` slice of the state, that is the source of truth of this value. Also any changes here should be reflected in the `meta` slice. So currently, the topBar state slice is redundant.

Reply:
All topBarContext removed for now as there is everything already specified.